assets/components/contribute/contribute.jsx

```diff
@@ -22,7 +21,6 @@ export default function Contribute(props: PropTypes) {
   const paymentImages = (
     <div className="component-contribute__payment-images">
       <Secure modifierClass="contribute-header" />
-      <InlinePaymentLogos />
     </div>
   );
```

Comment:
I think `paymentImages` can go away now, it was just there to wrap the two components used here. Now that only `Secure` is used you could pass it directly as `headingChildren` on line 34.